### PR TITLE
BAEL-4648: How to debug Websockets

### PR DESCRIPTION
**Pull request by navuluri**
Added code for BAEL-4648: How to debug Websockets.
Includes an integration test

### CODE REVIEW

File: spring-websockets/src/main/java/com/baeldung/debugwebsockets/StompWebSocketClient.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    public static void main(String[] args) {
+
+        WebSocketClient client = new StandardWebSocketClient();
+        WebSocketStompClient stompClient = new WebSocketStompClient(client);
+        stompClient.setMessageConverter(new MappingJackson2MessageConverter());
+        StompSessionHandler sessionHandler = new StompClientSessionHandler();
+        stompClient.connect(URL, sessionHandler);
+
+        new Scanner(System.in).nextLine();
+
```
Comment (KevinGilmore) [inline spring-websockets/src/main/java/com/baeldung/debugwebsockets/StompWebSocketClient.java:24]: As per guidelines, let's remove the empty lines at the beginning and end of this block

File: spring-websockets/src/test/java/com/baeldung/debugwebsockets/WebsocketIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+                            assertThat(payload).isNotNull();
+                            assertThat(payload).isInstanceOf(Map.class);
+
+                            @SuppressWarnings("unchecked")
+                            Map<String, Integer> map = (Map<String, Integer>) payload;
+
+                            assertThat(map).containsKey("HPE");
+                            assertThat(map.get("HPE")).isInstanceOf(Integer.class);
+
```
Comment (KevinGilmore) [inline spring-websockets/src/test/java/com/baeldung/debugwebsockets/WebsocketIntegrationTest.java:87]: Remove empty lines at beginning and end of block

**Review by KevinGilmore — COMMENTED** [review]
There are a couple of code blocks that have empty lines at the beginning and end (after the opening curly brace and before the closing one).

File: spring-websockets/src/main/java/com/baeldung/debugwebsockets/StompWebSocketClient.java (hunk 1 above)
Comment (navuluri) [reply spring-websockets/src/main/java/com/baeldung/debugwebsockets/StompWebSocketClient.java]: Thank You @KevinGilmore.  Made the change

File: spring-websockets/src/test/java/com/baeldung/debugwebsockets/WebsocketIntegrationTest.java (hunk 2 above)
Comment (navuluri) [reply spring-websockets/src/test/java/com/baeldung/debugwebsockets/WebsocketIntegrationTest.java]: Thank You @KevinGilmore.  Made the change

**Pull request merged**